types/stripe-js/elements/shipping-address.d.ts

```diff
@@ -122,6 +122,32 @@ export type StripeShippingAddressElement = StripeElementBase & {
     eventType: 'loaderstart',
     handler?: (event: {elementType: 'shippingAddress'}) => any
   ): StripeShippingAddressElement;
+
+  /**
+   * Updates the options the `ShippingAddressElement` was initialized with.
+   * Updates are merged into the existing configuration.
+   */
+  update(
+    options: StripeShippingAddressElementUpdateOptions
```

Comment:
nit: we can probably just do `options: Partial<StripeShippingAddressElementOptions>` here